apps/fabric-website/src/components/Nav/Nav.tsx

```diff
@@ -25,11 +24,9 @@ export class Nav extends React.Component<INavProps, INavState> {
       : null;
 
     return (
-      <FocusZone direction={ FocusZoneDirection.vertical } >
-        <nav className={ styles.nav } role='navigation'>
-          { links }
-        </nav>
-      </FocusZone>
+      <nav className={ styles.nav } role='navigation'>
```

Comment:
If this needs to be removed now due to blocking errors, we should definitely try to make sure it's restored in the future--users should be able to navigate the leftnav with arrow keys